Vic2ToHoI4/Source/Hoi4Outputter/Hoi4CountryOutputter.cpp

```diff
@@ -490,33 +494,56 @@ void outputPuppets(
 	const std::string& tag,
 	const std::string& governmentIdeology,
 	const std::set<std::string>& puppets,
-	const std::string& puppetMaster
+	const std::string& puppetMaster,
+	const std::map<std::string, HoI4Relations>& relations,
+	const bool& greatPower
 ) {
-	if (puppets.size() > 0)
+	if (puppets.size() > 0 || greatPower)
 	{
 		output << "# DIPLOMACY\n";
 		output << "if = {\n";
-		output << "    limit = {\n";
-		output << "        has_dlc = \"Together for Victory\"\n";
-		output << "    }\n";
-		for (const auto& puppet: puppets)
+		output << "\tlimit = {\n";
+		output << "\t\tOR = {\n";
+		output << "\t\t\thas_dlc = \"Together for Victory\"\n";
+		output << "\t\t\thas_dlc = \"Man the Guns\"\n";
+		output << "\t\t}\n";
+		output << "\t}\n";
+		if (puppets.size() > 0)
```

Comment:
This check is unnecessary. If puppets.size() == 0, the loop will never be entered anyways.